### PR TITLE
Fix for guildwars2.com

### PR DESCRIPTION
**Pull request by brybrant**
Fixed the header + navigation ; it was too bright

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -9342,6 +9342,27 @@ CSS
...
 
+guildwars2.com
+
+CSS
+body {
+    background-image: none !important;
+}
+body:before {
+    width: 100% !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9353]: I'm not sure about this particular change, seems like this isn't necessary. It doesn't obstruct any text.

Comment (brybrant) [reply src/config/dynamic-theme-fixes.config:9353]: Yes, it's not necessary but that background image on <body> is very bright for dark mode

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:9353]: Then we should remove the background image.

Comment (brybrant) [reply src/config/dynamic-theme-fixes.config:9353]: Do I need to make any changes to this pr?

**Pull request closed without merging**